packages/platforms/browser/lib/auto-instrumentation/page-load-phase-spans.ts

```diff
@@ -0,0 +1,73 @@
+import type { SpanFactory } from '@bugsnag/core-performance'
+import { type PerformanceWithTiming } from '../on-settle/load-event-end-settler'
+
+function shouldOmitSpan (startTime: number, endTime: number) {
+  return startTime === 0 && endTime === 0
+}
+
+export const instrumentPageLoadPhaseSpans = (spanFactory: SpanFactory, route: string, performance: PerformanceWithTiming) => {
+  const entry = performance.getEntriesByType('navigation')[0] as PerformanceNavigationTiming
+
+  if (entry) {
+    if (!shouldOmitSpan(entry.unloadEventStart, entry.unloadEventEnd)) {
+      spanFactory.endSpan(spanFactory.startSpan('[PageLoadPhase/Unload]' + route, {
+        startTime: entry.unloadEventStart
+      }), entry.unloadEventEnd)
+    }
+
+    if (entry.redirectStart && !shouldOmitSpan(entry.redirectStart, entry.redirectEnd)) {
```

Comment:
I _thought_ this value might not be present - but it will always be there, just `0` if no redirect occured.

```suggestion
    if (!shouldOmitSpan(entry.redirectStart, entry.redirectEnd)) {
```